src/scripts/oauth_tables_script.sql

```diff
@@ -0,0 +1,110 @@
+
+DROP TABLE IF EXISTS app_clients CASCADE;
+
+CREATE TABLE app_clients(
+             id BIGSERIAL NOT NULL,
+             client_name VARCHAR(50) UNIQUE NOT NULL,
+             client_id VARCHAR(50) UNIQUE NOT NULL,
+             client_secret VARCHAR(50) UNIQUE NOT NULL,
+             scope_id BIGINT NOT NULL,
+             expiration_time TIMESTAMP WITHOUT TIME ZONE,
+             entity_status status_enum NOT NULL,
+             created_by BIGINT NOT NULL,
+             created_date TIMESTAMP WITHOUT TIME ZONE,
+             updated_by BIGINT NOT NULL,
+             updated_date TIMESTAMP WITHOUT TIME ZONE,
+             PRIMARY KEY (id)
+);
+
+DROP TYPE IF EXISTS scope_type_enum CASCADE;
+
+CREATE TYPE scope_type_enum AS ENUM ('READ', 'WRITE', 'READ/WRITE');
+
+DROP TABLE IF EXISTS scopes CASCADE;
+
+CREATE TABLE app_scopes(
```

Comment:
If you are going for the table approach you need to a `app_clients_app_scopes` table to relate app clients with scopes

Reply:
done. table was remove.